docker-compose.yml

```
@@ -0,0 +1,59 @@
+version: "3.8"
+networks:
+  mapstore-network:
+    driver: bridge
+
+services:
+  db:
```

Rename database service
```suggestion
  postgres:
```